includes/template-library/sources/local.php

```diff
@@ -1283,6 +1281,8 @@ public function maybe_render_blank_state( $which ) {
 			return;
 		}
 
+		$additional_inline_style = '';
```

Comment:
Move the PR to `develop`.

Reply:
Handled: https://github.com/elementor/elementor/pull/13037